src/fluree/json_ld/impl/context.cljc

```diff
@@ -2,7 +2,8 @@
   (:require [fluree.json-ld.impl.iri :as iri]
             [fluree.json-ld.impl.util :refer [try-catchall]]
             [fluree.json-ld.impl.external :as external]
-            [clojure.string :as str]))
+            [clojure.string :as str])
+  (:import (java.net URI)))
```

Comment:
This is a .cljc file, so this approach won't work.

Reply:
Just needed a CLJS implementation too. Added in [f4ba92a](https://github.com/fluree/json-ld/pull/23/commits/f4ba92a6ad7ea7e246786b0319d8d6bf17ce5ddb)